test/e2e_node/pod_resize_test.go

```diff
@@ -1303,7 +1303,7 @@ func doPodResizeErrorTests() {
 //       Above tests are performed by doSheduletTests() and doPodResizeResourceQuotaTests()
 //       in test/node/pod_resize_test.go
 
-var _ = SIGDescribe("Pod InPlace Resize Container", framework.WithSerial(), feature.InPlacePodVerticalScaling, func() {
+var _ = SIGDescribe("Pod InPlace Resize Container", framework.WithSerial(), feature.InPlacePodVerticalScaling, "[NodeAlphaFeature:InPlacePodVerticalScaling]", func() {
```

Comment:
not so long ago we were discussing `NodeFeature` vs `Feature` but I don't recall what we decided. Perhaps @SergeyKanzhelev does?